### ISSUE TITLE
Openstack Provider crash in 0.6.11

### ISSUE THREAD

**Issue by krjensen** [hashicorp/terraform#4954]
[WINDOWS]

After upgrading to 0.6.11 my terraforms just crash, if i'm rolling back to 0.6.10 it works again. 

[crash.txt](https://github.com/hashicorp/terraform/files/114200/crash.txt)

**Comment by jen20** [comment]
Hi @krjensen! Thanks for reporting this - from the log you've posted this doesn't look like a Windows problem - it's the openstack provider:

```
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: panic: interface conversion: interface is nil, not bool
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: 
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: goroutine 70 [running]:
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: github.com/hashicorp/terraform/builtin/providers/openstack.getInstanceAccessAddresses(0xc082274ba0, 0xc08219cc90, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0)
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: 	/opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/openstack/resource_openstack_compute_instance_v2.go:1001 +0x437
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: github.com/hashicorp/terraform/builtin/providers/openstack.resourceComputeInstanceV2Read(0xc082274ba0, 0xb2e6a0, 0xc0822942c0, 0x0, 0x0)
2016/02/02 16:47:22 [DEBUG] terraform-provider-openstack.exe: 	/opt/gopath/src/github.com/hashicorp/terraform/builtin/providers/openstack/resource_openstack_compute_instance_v2.go:493 +0x643
```

It looks like df660a26a was responsible for the code that panic'd here. We will investigate this and fix.

**Comment by jtopjian** [comment]
@krjensen Can you post a sanitized version of your terraform configuration?

**Comment by jtopjian** [comment]
Actually, I think I already see the issue. Sorry about this.

If you're able to, can you confirm if #4962 fixes the problem?

**Comment by jtopjian** [comment]
hrm... yes, if you're able to, can you provide a sanitized version of your configuration as well as the steps to reproduce this? I'm unable to reproduce this by creating instances using Terraform 0.6.10 and then refreshing / re-applying with Terraform 0.6.11.

If I can reproduce this issue on my own, it'd help me not make this mistake in the future :smile:

**Comment by jtopjian** [comment]
Arg. I'm totally off my game today.

Somehow I was able to start reproducing this issue easily. I've closed #4962 in favor of #4966 since it's more Go-ish. I can confirm this fixes the problem.
